ArduPlane/GCS_Mavlink.cpp

```diff
@@ -162,24 +162,24 @@ void GCS_MAVLINK_Plane::send_nav_controller_output() const
 
         mavlink_msg_nav_controller_output_send(
             chan,
-            targets.x * 1.0e-2f,
-            targets.y * 1.0e-2f,
-            targets.z * 1.0e-2f,
+            targets.x * 0.01,
+            targets.y * 0.01,
+            targets.z * 0.01,
             wp_nav_valid ? quadplane.wp_nav->get_wp_bearing_to_destination() : 0,
-            wp_nav_valid ? MIN(quadplane.wp_nav->get_wp_distance_to_destination(), UINT16_MAX) : 0,
-            (plane.control_mode != &plane.mode_qstabilize) ? quadplane.pos_control->get_pos_error_z_cm() * 1.0e-2f : 0,
+            wp_nav_valid ? MIN(quadplane.wp_nav->get_wp_distance_to_destination() * 0.01, UINT16_MAX) : 0,
+            (plane.control_mode != &plane.mode_qstabilize) ? quadplane.pos_control->get_pos_error_z_cm() * 0.01 : 0,
```

Comment:
More stuff that is not your problem. 
```suggestion
            quadplane.pos_control->is_active_z() ? quadplane.pos_control->get_pos_error_z_cm() * 0.01 : 0,
```
We have Qacro as a second Q mode that does not use the Z controller, so this old check for only Qstab is now wrong.

Reply:
Looks like we make the same mistake in `Log_Write_QControl_Tuning` and `high_latency_target_altitude`

Reply:
@stephendade on the high-latency thing